src/component/tooltip/TooltipModel.ts

```diff
@@ -46,6 +46,10 @@ export interface TooltipOption extends CommonTooltipOption<TopLevelFormatterPara
      * If show popup content
      */
     showContent?: boolean
+    /**
+     * escape html, prevent xss
+     */
+    escapeContent?: boolean
```

Comment:
I think we should neither add this option, nor `encodeHTML` internally.
The reason is:
+ See the [comment](https://github.com/apache/echarts/issues/14429#issuecomment-794847914), users should do the encode themselves if the assemble html snippet in `tooltip.formatter`.
+ If echarts do the "encodeHTML" internally by this option `escapeContent`, echarts should not only do `encodeHTML` to part of the params but left other params not encoded. Otherwise users will be confused.
But for some complex data structure like `params.data`, it's not an easy work to encode it.

it's not easy to do it completely.